src/fragment-components/a-inline-loading.tsx

```diff
@@ -0,0 +1,256 @@
+		},
+		finished: {
+			iconDescription: componentObj.finishedIconDescription,
+			description: componentObj.successText
+		}
+	};
+
+	return (
+		<AComponent
+		componentObj={componentObj}
+		headingCss={css`display: block;`}
+		rejectDrop={true}
+		{...rest}>
+			<InlineLoading
+			successDelay={componentObj.successDelay}
+			description={status[componentObj.status].description}
+			iconDescription={status[componentObj.status].iconDescription}
+			status={componentObj.status}
+			className={componentObj.cssClasses?.map((cc: any) => cc.id).join(' ')} />
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: AInlineLoading,
+	codeUI: AInlineLoadingCodeUI,
+	settingsUI: AInlineLoadingSettingsUI,
+	keywords: ['inline', 'loading'],
+	name: 'Inline Loading',
+	type: 'inline-loading',
+	defaultComponentObj: {
+		type: 'inline-loading',
+		status: 'active',
+		activeText: '',
+		inactiveText: '',
+		successText: '',
+		errorText: '',
+		activeIconDescription: '',
+		errorIconDescription: '',
+		finishedIconDescription: '',
+		inactiveIconDescription: '',
+		successDelay: 1500
+	},
+	image,
+	codeExport: {
+		angular: {
+			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Status = "${json.status}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}LoadingText = "${json.activeText}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}SuccessText = "${json.successText}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}ErrorText = "${json.errorText}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}SuccessDelay = ${json.successDelay};`,
+			outputs: ({ json }) => `@Output() ${nameStringToVariableString(json.codeContext?.name)}OnSuccess = new EventEmitter();`,
+			imports: ['InlineLoadingModule'],
+			code: ({ json }) => {
+				return `<ibm-inline-loading
+					(onSuccess)="${nameStringToVariableString(json.codeContext?.name)}OnSuccess.emit($event)"
+					[successDelay]="${nameStringToVariableString(json.codeContext?.name)}SuccessDelay"
+					[state]="${nameStringToVariableString(json.codeContext?.name)}Status"
+					[loadingText]="${nameStringToVariableString(json.codeContext?.name)}LoadingText"
+					[successText]="${nameStringToVariableString(json.codeContext?.name)}SuccessText"
+					[errorText]="${nameStringToVariableString(json.codeContext?.name)}ErrorText"
+					${angularClassNamesFromComponentObj(json)}>
+				</ibm-inline-loading>`;
+			}
+		},
+		react: {
+			imports: ['InlineLoading'],
+			code: ({ json }) => {
+				const status = `state["${nameStringToVariableString(json.codeContext?.name)}"] || "${json.status}"`;
+				return `<InlineLoading
+					onSuccess={() => {
+						if(typeof state.${nameStringToVariableString(json.codeContext?.name)}OnSuccess === "function") {
+							return state.${nameStringToVariableString(json.codeContext?.name)}OnSuccess;
+						}
+						return undefined;
+					})}
```

Comment:
```suggestion
					}}
```

Incorrect syntax on export for react